### DISCUSSION TITLE
Enter time in format 11.53 (change the timeseparator from : to . )

### DISCUSSION THREAD

**Discussion by sandersemmel** [primefaces/community#872] [PrimeNG] [answered]
When using p-calendar

You can set the dateformat for the calendar like so:

dateFormat="hh:mm"

In this case you can enter into the input box 11:53 and after changing focus the text stays in the box.

However, I would like to enter 11.53 instead.

I tried this:

dateFormat="hh.mm"

If I enter 11.53, the entered value does not stay in the input box. Is it possible to use this kind of syntax ?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Unfortunately, dateFormat is supported by date input, not time input.
